vocode/streaming/streaming_conversation.py

```diff
@@ -1465,7 +1472,9 @@ async def send_speech_to_output(
         # Added stop event check otherwise it will block other synthesis result tasks
         # even though we meant to cancel this one.
         sleep_interval = 0.1  # Mark last action every 0.1 seconds
-        remaining_sleep = total_time_sent
+        remaining_sleep = total_time_sent - self.get_duration_spoken_seconds(
```

Comment:
this sure looks like it will affect behavior

Reply:
Depends on what your definition of behavior is. Yes we are changing that value. Will it affect our conversation mechanics enough to modify how our agents respond to the same human input. I don't think so but as I said in the description I need to test more and key word **_shouldn't_** modify existing behavior.